js/gamestate.js

```diff
@@ -113,12 +163,10 @@ class GameState {
 
     if (move.op == "beginTurn") {
       this.beginTurn()
-    } else if (move.op == "attack") {
-      this.attack(move.from, move.to)
-    } else if (move.op == "face") {
-      this.face(move.from)
-    } else if (move.op == "play") {
-      this.play(move.from)
+    } else if (move.op == "clickCard") {
```

Comment:
if we have a mobile thing then "click" won't be accurate. how about calling it "select" rather than "click", because it could also be a tap